cmd/erda-server/bootstrap.yaml

```diff
@@ -279,6 +282,7 @@ uc-adaptor:
 dop: { }
 erda.core.dop.taskerror: { }
 erda.dop.qa.unittest: { }
+project-report: { }
```

Comment:
We should unify the term. I think `project-report` may contains other metrics more than issue-management.
So we should use `project-management-report` here.

Reply:
adjusted